README.md

```diff
@@ -30,6 +32,13 @@ docker run --rm -ti \
   -p 8080:8080 \
   -e 'AWS_PROFILE=<SOME PROFILE>' \
   aws-sigv4-proxy -v
+
+# Stripping out old Sigv2 authorization headers
+docker run --rm -ti \
+  -v ~/.aws:/root/.aws \
+  -p 8080:8080 \
+  -e 'AWS_PROFILE=<SOME PROFILE>' \
+  aws-sigv4-proxy -v -s Authorization
```

Comment:
I like having this example - can you add this specific example to the #Examples section and leave the default case here? :)

Reply:
I'm slightly confused on this one.  The examples all seem to be examples of using curl to interact with the running service, where as this section is about different ways to invoke the server.  I'm happy to move this anywhere, just slightly confused as to what you want me to do :)

Reply:
Whoops! Totally my bad - ignor my previous comment :) GTG! :shipit: